src/snowcli/plugins/nativeapp/version/commands.py

```diff
@@ -41,19 +41,19 @@ def create(
     version: Optional[str] = typer.Argument(
         None,
         help=f"""The identifier or 'version string' of the version you would like to create a version and/or patch for.
-        Defaults to undefined, which means the CLI will use the version, if present, in the manifest.yml.""",
+        Defaults to undefined, which means the CLI uses the version, if present, in the `manifest.yml`.""",
     ),
     patch: Optional[str] = typer.Option(
         None,
         "--patch",
         help=f"""The patch number you would like to create for an existing version.
-        Defaults to undefined if it is not set, which means the CLI will either use the version, if present, in the manifest.yml,
-        or auto-generate the patch number.""",
+        Defaults to undefined if it is not set, which means the CLI either uses the patch, if present, in the `manifest.yml`,
```

Comment:
Defaults to undefined if it is not set, which means the CLI either uses the patch specified in the `manifest.yml` file or automatically generates a new patch number.